q2_vsearch/_join_pairs.py

```diff
@@ -136,6 +139,8 @@ def _join_pairs_w_command_output(
             cmd += ['--fastq_maxmergelen', str(maxmergelen)]
         if maxee is not None:
             cmd += ['--fastq_maxee', str(maxee)]
+        if threads is not None:
```

Comment:
Since the default is set to `1`, QIIME 2 will not allow `None` to be passed. (`None` is only "in the type" if it is set as the default value). So you can drop the `if` statement!